### PR TITLE
chore: sanitize AAVE sources

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

File: packages/oracle-node/manifests/dev/main-wide-support.json [hunk 1]
```diff
@@ -683,30 +683,30 @@
...
-        "chainlink",
+        "bybit",
         "coinbaseprime",
         "coinbasepro",
         "coingecko",
         "currencycom",
-        "dia",
         "hitbtc",
         "huobipro",
-        "kaiko",
```
Comment (koonopek) [inline packages/oracle-node/manifests/dev/main-wide-support.json:701]: Why do we remove kaiko?

Comment (koonopek) [reply packages/oracle-node/manifests/dev/main-wide-support.json:701]: Hmm why do remove some of the sources?

Comment (koonopek) [reply packages/oracle-node/manifests/dev/main-wide-support.json:701]: image
For me not checked sources from picture above have very deviated prices

**Review by koonopek — DISMISSED** [review]
Please answer one comment, just for my better understanding

File: packages/oracle-node/manifests/dev/main-wide-support.json (hunk 1 above)
Comment (cehali) [reply packages/oracle-node/manifests/dev/main-wide-support.json:701]: Chainlink and Cex are removed in this PR. Binance is not working correctly in the US, we will move our infra to Europe. Bittrex was working correctly when applying those changes, we need to monitor it. Kaiko should be there, my bad, will fix it in a moment. The rest of the removed sources were not providing bad values or they are other oracles, which we don't want to use in more prod env.

**Review by koonopek — DISMISSED** [review]

**Pull request merged**